project-management.md

```diff
@@ -0,0 +1,35 @@
+# Project Management
+
+The OpenTelemetry has limited bandwith for managing changes which expand the scope of OpenTelemetry, or impact many SIGs within OpenTelemetry. There are three common scenarios which have this kind of impact.
+* Non-trivial changes to the OpenTelemetry specification.
+* A new SIG being formed.
+* An existing SIG taking on new work which will affect the OpenTelemetry as a whole, and will need review from the broader community.
+
+Any changes which fall into one of the above categories must first create a project proposal and gain approval from the GC and TC before work begins.
+
+## Project Proposal
+
+At minimum, projects require the following resources to be successful:
+
+* A clearly defined set of goals and deliverables.
+* Deadlines for when the deliverables will be ready for review by the broader community.
```

Comment:
Should these be part of the initial proposal or can these be laid out once the group has formed and work is starting?